core/server/controllers/frontend.js

```diff
@@ -564,8 +565,9 @@ frontendControllers = {
                                 // make sure URL has a trailing slash
                                 try {
                                     parsed = url.parse(attributeValue);
+                                    path = (parsed.pathname) ? parsed.pathname : null;
 
-                                    if (parsed.pathname && parsed.pathname.slice(-1) !== '/') {
+                                    if (path && path.slice(-1) !== '/' && path.indexOf('content/images/') === -1) {
```

Comment:
> I am not sure that we need to add slashes to these urls at all

That's a good point.  The cost of an occasional redirect is probably worth the simplification we get by not adding trailing slashes at all.